.github/workflows/sync-translated-content.yml

```
@@ -33,25 +47,20 @@ jobs:
 
       - name: Install all yarn packages
         working-directory: ${{ github.workspace }}/mdn/content
-        run: |
-          yarn --frozen-lockfile
+        run: yarn --frozen-lockfile
 
       - name: Sync translated content
         env:
           CONTENT_ROOT: ${{ github.workspace }}/mdn/content/files
           CONTENT_TRANSLATED_ROOT: ${{ github.workspace }}/files
         working-directory: ${{ github.workspace }}/mdn/content
-        run: |
-          yarn content sync-translated-content
-
-      - name: Commit changes
-        # git commit will fail if there are no changes but that's okay!
-        continue-on-error: true
-        run: |
-          cd $GITHUB_WORKSPACE
-          git remote add upstream "https://${GITHUB_ACTOR}:${GITHUB_TOKEN}@github.com/${GITHUB_REPOSITORY}.git"
-          git config --local user.email "github-actions[bot]@users.noreply.github.com"
-          git config --local user.name "github-actions[bot]"
-          git commit -a -m "[CRON] sync translated content" && \
-          git pull --rebase upstream main && \
-          git push upstream main
+        run: yarn content sync-translated-content ${{ matrix.lang }}
+
+      - name: Create PR with sync for ${{ matrix.lang }}
+        uses: peter-evans/create-pull-request@v4
+        with:
+          commit-message: "${{ matrix.lang }}: sync translated content"
+          branch: content-sync-${{ matrix.lang }}
+          title: "[${{ matrix.lang }}] sync translated content"
+          author: mdn-bot <108879845+mdn-bot@users.noreply.github.com>
```

Should PR's `author` be the same as the `committer`? (Or when we use `Squash and merge`, the commit to main branch will be [`Co-authored`](https://docs.github.com/en/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors))

And we have used `github-actions[bot]` in [previous content-sync commit](https://github.com/mdn/translated-content/commits?author=github-actions%5Bbot%5D). Maybe use `github-actions[bot] github-actions[bot]@users.noreply.github.com` is better:

```suggestion
          commit-message: "${{ matrix.lang }}: sync translated content"
          committer: github-actions[bot] <github-actions[bot]@users.noreply.github.com>
          branch: content-sync-${{ matrix.lang }}
          title: "[${{ matrix.lang }}] sync translated content"
          author: github-actions[bot] <github-actions[bot]@users.noreply.github.com>
```

@caugner What's your opinion?

---

Yes, for consistency, it probably makes sense to use `github-actions[bot] <github-actions[bot]@users.noreply.github.com>`, but `author` should suffice (like [here](https://github.com/mdn/content/blob/e47cebe4a6db41979dcb37ac7115b38513f8b873/.github/workflows/interfacedata-updater.yml#L35-L45)).

```suggestion
          commit-message: "${{ matrix.lang }}: sync translated content"
          branch: content-sync-${{ matrix.lang }}
          title: "[${{ matrix.lang }}] sync translated content"
          author: github-actions[bot] <github-actions[bot]@users.noreply.github.com>
```